### PR TITLE
Updating docs

### PR DESCRIPTION
**Pull request by acholyn**
Docs have been streamlined to reflect the current state and updated with information on how the email notifications were set up. Some files have been renamed (how to set up and aws base settings)

### CODE REVIEW

**Review by p-j-smith — APPROVED** [review]
Looks great! A few questions for my understanding, but nothing blocking

File: README.md [hunk 1]
```diff
@@ -4,13 +4,13 @@ The production website is hosted at http://museumofdreamworlds.eu-west-2.elastic
 
 The staging one is http://museumofdreams.eu-west-2.elasticbeanstalk.com/ and is built from the `development` branch
 
-The more public versions are https://museumofdreamworlds.org and https://staging.museumofdreamworlds.org
+The offical versions are https://museumofdreamworlds.org and https://staging.museumofdreamworlds.org
```
[the remark this answers is not in the document]
Comment (acholyn) [reply README.md:7]: Yep, the official versions are just with domains we've purchased so they sort of lie on top of the aws ones

File: README.md [hunk 2]
```diff
@@ -4,13 +4,13 @@ The production website is hosted at http://museumofdreamworlds.eu-west-2.elastic
...
 
-Development should be done locally and staged to the website. At present this is not a production version.
+Development should be done locally and pushed to the staging website where researchers can test features and sign off on them before putting the code on production.
```
[the remark this answers is not in the document]
Comment (acholyn) [reply README.md:9]: Through PR usually, should I make that more explicit?

File: docs/AWSses.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
+# Setting up email notification system for changes on the website
+
+These instructions assume you have purchased a domain name.
+
+I followed the instructions in [this article](https://medium.com/hackernoon/the-easiest-way-to-send-emails-with-django-using-ses-from-aws-62f3d3d33efd). You don't need the seciton on sending from personal emails, but do need the section sending from a domain (that you've purchased).
+
+When you have set up a domain in SES, you need to ensure you have selected "publish to Route 53" so that the details (DKIM) get added to the relevant domain name. You don't need to set up the `Custom MAIL FROM domain` unless you want to (allows you to send from abc.yourdomain), if you do, ensure you publish those credentials to Route 53.
```
[the remark this answers is not in the document]
Comment (acholyn) [reply docs/AWSses.md:7]: nope, using the normal domain is sufficient for us

File: docs/awsSSL.md [hunk 4]
```diff
@@ -18,4 +18,4 @@ Go to the EB environment, go to the `Configuration` tab and go to the section ab
 
 Once the changes have been applied, go to your `EC2 Security Groups` and here you should see a new load balancer type for your environment. I recommend changing the name to include LB or something similar so you can tell them apart easily.
 
-On your environment's security group, the inbound rules should include http traffic from the load balancer, add it if it doesn't. Update the inbound rules in the load balancer's security group to allow HTTPS traffic from: itself, 0.0.0.0 (anywhere), the database and the environment. Also allow HTTP from anywhere.
+On your environment's security group, the inbound rules should include http traffic from the load balancer, add it if it doesn't. Update the inbound rules in the load balancer's security group to allow HTTPS traffic from 0.0.0.0 (anywhere), the database and the environment. Also allow HTTP from anywhere.
```
[the remark this answers is not in the document]
Comment (acholyn) [reply docs/awsSSL.md:21]: it's to the load balancer, but on AWS things go both ways so technically it's also from the load balancer too (but we care about the incoming traffic more)

File: .ebextensions/db-migrate.config [hunk 5]
```diff
@@ -1,11 +0,0 @@
...
-  01_migrate_and_collectstatic:
-    test: "[ -f /var/app/current/manage.py ]"
-    command: "python manage.py migrate && python manage.py collectstatic --no-input && echo db-migrate has run"
-    leader_only: true
-    ignoreErrors: true
-option_settings:
-  aws:elasticbeanstalk:application:environment:
-    DJANGO_SETTINGS_MODULE: museum_of_dreams_project.settings.aws
-  aws:elasticbeanstalk:environment:proxy:staticfiles:
-    /static: static
```
[the remark this answers is not in the document]
Comment (acholyn) [reply .ebextensions/db-migrate.config:11]: it's handled by a script in `.platform`; originally we didn't know which way would work so had it in both but now it seems `.platform` works better

**Pull request merged**